.github/workflows/salesloft.yml

```diff
@@ -0,0 +1,75 @@
+name: SalesLoft
+on:
+  push:
+    branches: [ master ]
+    tags: [ 'v*' ]
+  pull_request:
+
+jobs:
+  rspec:
+    name: RSpec
+    runs-on: ubuntu-latest
+    services:
+      postgres:
+        image: postgres:10.10
+        env:
+          POSTGRES_USER: postgres
+          POSTGRES_PASSWORD: postgres
+          POSTGRES_DB: postgres
+        ports:
+          - 5432:5432
+        options: --health-cmd pg_isready --health-interval 10s --health-timeout 5s --health-retries 5
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - rails: "7.0"
+            ruby: 3.2
+          - rails: 6.1
+            ruby: 2.7
+          - rails: 6.1
+            ruby: 3.2
+
+          - rails: 5.2
+            ruby: 2.7
+
+
+    env:
+      RAILS_ENV: test
+      DATABASE_URL: postgresql://postgres:postgres@localhost/postgres
+      BUNDLE_GEMS__SALESLOFT__COM: ${{ secrets.SL_GEMSERVER_URL}}
+      MTSR_RAILS_VERSION: rails_${{ matrix.rails }}
+      DISPLAY: ":99.0"
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+          # runs 'bundle install' and caches installed gems automatically
+          bundler-cache: true
+
+      - name: Gemfile
+        run: echo $BUNDLE_GEMFILE
+
+      - name: Ruby Version
+        run: ruby --version
+
+      - name: Run Tests
+        run: bundle exec rspec
+
```

Comment:
let's add another job called `rspecall`. This job will basically make all the rspec runs pass, and this single job will be the "required" job:
you should be able to copy-paste this from here:
https://github.com/SalesLoft/bussey/blob/b56e32b67a114befe5bf278d8ded8ddebb063272/.github/workflows/salesloft.yml#L32-L40